CMakeLists.txt

```diff
@@ -15,9 +15,6 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
 # OSTree is mandatory for aktualizr-lite
 option(BUILD_OSTREE "Set to ON to compile with ostree support" ON)
-option(BUILD_DOCKERAPP "Set to ON to compile with package manager support of docker-app" ON)
-# always build with DOCKERAPP for aktualizr-lite, optional only for aktualizr
-set(BUILD_DOCKERAPP ON)
```

Comment:
We still need some option to disable compose-apps. We don't have docker/golang on riscv yet.

Reply:
BUILD_DOCKERAPP has to be removed anyway as it doesn't manage the inclusion or exclusion of the compose app manager into a compilation.

>We don't have docker/golang on riscv yet

But it doesn't mean that the compose-apps package manager itself can be built on those platforms.

Reply:
true. i guess we just need to make sure we never set that package manager on those platforms. @ricardosalveti - How are we handling riscv related to this?